source/administration/identity-access-management/policy-based-access-control.rst

```diff
@@ -816,3 +816,218 @@ MinIO supports the following conditions for use with defining policies for
+
+         * - ``jwt:upn``
+           - Returns the User Principal Name claim from the client authentication information.
+
+         * - ``jwt:name``
+           - Returns the ``name`` claim for the user.
+
+         * - ``jwt:groups``
+           - Returns the ``groups`` claim for the user.
+
+         * - ``jwt:given_name``
+           - Returns the ``given_name`` claim for the user.
+
+         * - ``jwt:family_name``
+           - Returns the ``family_name`` claim for the user.
+
+         * - ``jwt:middle_name``
+           - Returns the ``middle_name`` claim for the user.
+
+         * - ``jwt:nickname``
+           - Returns the ``nickname`` claim for the user.
+
+         * - ``jwt:preferred_username``
+           - Returns the ``preferred_username`` claim for the user.
+
+         * - ``jwt:profile``
+           - Returns the ``profile`` claim for the user.
+
+         * - ``jwt:picture``
+           - Returns the ``picture`` claim for the user.
+
+         * - ``jwt:website``
+           - Returns the ``website`` claim for the user.
+
+         * - ``jwt:email``
+           - Returns the ``email`` claim for the user.
+
+         * - ``jwt:gender``
+           - Returns the ``gender`` claim for the user.
+
+         * - ``jwt:birthdate``
+           - Returns the ``birthdate`` claim for the user.
+
+         * - ``jwt:phone_number``
+           - Returns the ``phone_number`` claim for the user.
+
+         * - ``jwt:address``
+           - Returns the ``address`` claim for the user.
+
+         * - ``jwt:scope``
+           - Returns the ``scope`` claim for the user.
+
+         * - ``jwt:client_id``
+           - Returns the ``client_id`` claim for the user.
+   
+      See the `OpenID Connect Core 1.0 <https://openid.net/specs/openid-connect-core-1_0.html>` document for more information on these scopes.
+      Your OIDC provider of choice may have more specific documentation.
+
+      For example, the following policy uses variables to substitute the authenticated user's ``PreferredUsername`` as part of the ``Resource`` field such that the user can only access those prefixes which match their username:
+
+      .. code-block:: json
+
+         {
+         "Version": "2012-10-17",
+         "Statement": [
+               {
+                  "Action": ["s3:ListBucket"],
+                  "Effect": "Allow",
+                  "Resource": ["arn:aws:s3:::mybucket"],
+                  "Condition": {"StringLike": {"s3:prefix": ["${jwt:PreferredUsername}/*"]}}
+               },
+               {
+                  "Action": [
+                  "s3:GetObject",
+                  "s3:PutObject"
+                  ],
+                  "Effect": "Allow",
+                  "Resource": ["arn:aws:s3:::mybucket/${jwt:PreferredUsername}/*"]
+               }
+            ]
+         }
+
+      The ``${jwt:PreferredUsername}`` substitutes in the username to that portion of the policy.
```

Comment:
> substitutes in the username to that portion of the policy.
It took me a few tries to read this as intended. (There's also a similar line below, in the LDAP example.)

My understanding is these are template variables, replaced by a real value when known. Perhaps something like this?

> ``${jwt:PreferredUsername}`` in the ``Resource`` field is replaced with the authenticated user's name.

I feel like there could be "when the policy is applied" at the end too, although I'm not sure that is correct terminology.